pkg/command/autoscaling/update.go

```diff
@@ -28,13 +30,20 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
 
 	"knative.dev/client/pkg/kn/flags"
+
+	as "knative.dev/serving/pkg/apis/autoscaling"
 )
 
 var (
-	scaleToZero       bool
-	enableScaleToZero = "enable-scale-to-zero"
-	knativeServing    = "knative-serving"
-	configAutoscaler  = "config-autoscaler"
+	scaleToZero                                                         bool
+	enableScaleToZero                                                   = "enable-scale-to-zero"
```

Comment:
I suggest moving these to one or multiple structs... for instance:
```go
type config struct {
  ScaleToZero bool
  EnableScaleToZero bool
  ...
}
```

You can then manage all of these global with one instantiation of the `config` type. OR group the related ones into their own structs.

Reply:
@maximilien Thanks Max for your review. Comments addressed.